src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -540,6 +540,57 @@
   <data name="Event_Hookup_Command_Handler" xml:space="preserve">
     <value>Generate Event Subscription Command Handler</value>
   </data>
+  <data name="Remove_unnecessary_usings" xml:space="preserve">
+    <value>Remove unnecessary usings</value>
+  </data>
+  <data name="Sort_usings" xml:space="preserve">
+    <value>Sort usings</value>
+  </data>
+  <data name="Format_document_settings" xml:space="preserve">
+    <value>Format Document Settings (Experiment) </value>
+  </data>
+  <data name="All_csharp_formatting_rules" xml:space="preserve">
+    <value>All C# Formatting Rules (indentation, wrapping and spacing)</value>
```

Comment:
data name needs to be All_csharp_Formatting_Rules_Indentation_Wrapping_and_spacing to match value.

Reply:
➡️ We can verify all these during the strings pass at the end.